4.7pythagoras.py

```diff
@@ -0,0 +1,13 @@
+def pythagoras():
+    """finds the pythagoras triangles between 1 and 100."""
+    pythagoras_list = []
+    for a in range(1, 101):
+        for b in range(a, 101):
+            for c in range(b, 101):
+                if a**2+b**2 == c**2:
+                    triangle = (a, b, c)
+                    pythagoras_list.append(triangle)
+    return tuple(pythagoras_list)
+
+
+print(pythagoras())
```

Comment:
- kodlarınız çalışıyor. tebrikler.
- açıklama eklemeniz güzel olmuş.
aşağıya farklı bir çözüm eklenmiştir.

def pythagorean(limit):
    c = 0
    m = 2 # for range(1,m --> m-1)

    while c < limit:
        for n in range(1, m):
            a = (m * m) -(n * n)
            b = (2 * m * n)
            c = (m * m) + (n * n)
            if c > limit:
                break
            print(a, b, c)

        m = m + 1
limit = int(input("Enter a Limit : ")) # until limit it will write pythagorean tringles
pythagorean(limit)